site/docs/components/icons.md

```diff
@@ -114,40 +177,18 @@ The following brand-specific color classes are provided:
 {% endcapture %}
 {% include example.html content=example %}
 
-### Transform
 
-Transform icons with [transform utilities](/docs/utilities/transform)
+## Formatting a new SVG
 
-{% capture example %}
-<svg class="rotate-180">
-  <use xlink:href="#right_arrow" />
-</svg>
-{% endcapture %}
-{% include example.html content=example %}
+Each icon in the system is a formatted svg with a `viewBox="0 0 500 500"`, a path fill="currentColor" and a descriptive class name.
 
-## All Icons
+The SVG is created on a 500px x 500px art board. SVG's created in a design program such as Adobe Illustrator or Sketch contain extra information. Clean up an svg with this online tool https://jakearchibald.github.io/svgomg/. Finally format the svg to match the example below.
+ 
```

Comment:
extra information like what? Either reference it here or see my bullet points note below.